src/test/java/edu/cornell/kfs/module/purap/CuPurapTestConstants.java

```diff
@@ -26,4 +26,6 @@ public final class CuPurapTestConstants {
     public static final String XSI_NAMESPACE_URL = "http://www.w3.org/2001/XMLSchema-instance";
 
     public static final int REQUISITION_ITEM_DESCRIPTION_MAX_LENGTH = 254;
+    
+    public static final String JAGGAER_UPLOAD_SUPPLIERS_TEST_DTD_TAG = "<!DOCTYPE SupplierSyncMessage SYSTEM \"https://usertest-messages.sciquest.com/app_docs/dtd/supplier/TSMSupplier.dtd\">";

```

Comment:
In the running application, this tag value is pulled from a parameter.  In the test cases, this value is compared to a static text file.  It is not validated to anything external.  If or When Jaggaer changes their DTD locations, we will just need to update parameter values.
If Jaggaer changes the XML syntax in the DTD, we will need to make code changes to support that change anyway.  I don't think this will cause problems

Reply:
No, that won't happen that way.
The value of JAGGAER_UPLOAD_SUPPLIERS_TEST_DTD_TAG corresponds to a static XML testing file we have in the unit test package.  A developer would need both when a need arises.  If the functionals change the value of the parameter, it won't affect the unit tests.

We would need to change the parameter value if Jaggaer tells us they moved the DTD location.  If they do change the DTD location, I would expect that would also mean XML structural changes that would require further coding changes